.github/workflows/note-arduino-ci.yml

```diff
@@ -26,3 +26,40 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  validate_examples: # job id
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        example-sketch:
+          - ./examples/Example0_LibrarylessCommunication/Example0_LibrarylessCommunication.ino
+          - ./examples/Example1_NotecardBasics/Example1_NotecardBasics.ino
+          - ./examples/Example2_PeriodicCommunications/Example2_PeriodicCommunications.ino
+          - ./examples/Example3_InboundPolling/Example3_InboundPolling.ino
+          - ./examples/Example4_InboundInterrupts/Example4_InboundInterrupts.ino
+          - ./examples/Example5_UsingTemplates/Example5_UsingTemplates.ino
+          - ./examples/Example6_SensorTutorial/Example6_SensorTutorial.ino
+          - ./examples/Example7_PowerControl/Example7_PowerControl.ino
```

Comment:
I presume it's not possible to glob all examples in a directory instead of having to reference them all by name?

Reply:
This does not feed into a script, but is instead each permutation is provided directly to an image to compile a script.